cmd/mt-replicator/metrics.go

```diff
@@ -10,6 +11,11 @@ import (
 	"gopkg.in/raintank/schema.v1"
 )
 
+var (
+	metricsBatchSize    = flag.Int("metrics-batch-size", 1000, "number of metrics in each write batch")
+	metricsFetchDefault = flag.Int("metrics-fetch-defaut", 32768, "default number of bytes to fetch with each read")
```

Comment:
name is incorrect. bytes, not metrics. 
let's use the same terminology as kafkaMdm input:
```
input/kafkamdm/kafkamdm.go
56:var consumerFetchDefault int
78:	inKafkaMdm.IntVar(&consumerFetchDefault, "consumer-fetch-default", 32768, "The default number of message bytes to fetch in a request")
123:	config.Consumer.Fetch.Default = int32(consumerFetchDefault)
```